development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
```

Comment:
That `cask` [Qt Creator install](https://github.com/caskroom/homebrew-cask/blob/master/Casks/qt-creator.rb) does not link to anything in Homebrew, just automates the install of the DMG distro. It does not even link to the Creator download page.

Adding `brew cask` to the mix is a double-edged sword. Yes, it easily installs the app, but doesn't necessarily give you the latest, nor give one the opportunity to verify any code-signing certificates. I think it best to point users to the real download location for app installs, so they can also read the changelog and see any other information that the distributor wishes to convey.